src/psyclone/psyir/symbols/datasymbol.py

```diff
@@ -342,21 +342,24 @@ def unresolved_interface(self):
     @constant_value.setter
     def constant_value(self, new_value):
         '''
-        :param constant_value: Set or change the fixed known value of \
-        the constant for this DataSymbol. If the value is None then this \
-        symbol does not have a fixed constant. The datatype of \
-        new_value must be compatible with the datatype of the symbol.
-        :type constant_value: int, str or bool
-
-        :raises ValueError: If a non-None value is provided and 1) \
-        this DataSymbol instance does not have local scope, or 2) this \
-        DataSymbol instance is not a scalar (as the shape attribute is not \
-        empty), or 3) a constant value is provided but the type of the \
-        value does not support this, or 4) the type of the value \
-        provided is not compatible with the datatype of this DataSymbol \
-        instance.
-
-        '''
+        :param constant_value: Set or change the fixed known value of the \
```

Comment:
s/constant_value/new_value/ same for the the :type: below.

Reply:
Done